### DISCUSSION TITLE
cant open on my rog2 phone

### DISCUSSION THREAD

**Discussion by EnSteav** [aivanovski/keepassvault#230] [General]
like i said , it cant open.

**Comment by EnSteav** [comment]
its android 10

**Reply by EnSteav** [reply]
i mean the app quit immediately after i open the app. anyway ,its ok now. thanks,   i need this app so badly.

**Comment by aivanovski** [comment]
Hi,
Please provide more clear explanation. It's not clear what are you going to do.

**Reply by EnSteav** [reply]
its fixed after i update. lol
